bca4abm/processors/link.py

```diff
@@ -150,6 +150,10 @@ def eval_link_spec(link_spec, link_file_names, data_dir,
         results[scenario] = summary
 
         if trace_results is not None:
+            # FIXME: manually setting df.index.name to prevent
```

Comment:
`tracing.write_csv` is in asim. It checks if `df.index.name in df` and resets the index if not. This particular df uses a multi-index though, so this check incorrectly fails.

Reply:
Did you create the appropriate issue in activitysim?

Reply:
done!

https://github.com/ActivitySim/activitysim/issues/269